src/app/pages/send-tokens/components/asset-search/asset-search.tsx

```diff
@@ -2,55 +2,69 @@ import { memo, useEffect, useState } from 'react';
 import { useField } from 'formik';
 import { Box, color, Stack } from '@stacks/ui';
 
-import { AssetWithMeta } from '@app/common/asset-types';
-import { useTransferableAssets } from '@app/store/assets/asset.hooks';
-import { useSelectedAsset } from '@app/pages/send-tokens/hooks/use-selected-asset';
+import {
+  useStacksCryptoCurrencyAssetBalance,
+  useTransferableStacksFungibleTokenAssetBalances,
+} from '@app/query/stacks/balance/crypto-asset-balances.hooks';
+import { useSelectedAssetBalance } from '@app/pages/send-tokens/hooks/use-selected-asset-balance';
 import { useCurrentAccountAvailableStxBalance } from '@app/query/stacks/balance/balance.hooks';
+import type {
+  StacksCryptoCurrencyAssetBalance,
+  StacksFungibleTokenAssetBalance,
+} from '@shared/models/crypto-asset-balance.model';
 
 import { AssetSearchField } from './asset-search-field';
 import { SelectedAsset } from './selected-asset';
 
-function principalHasOnlyOneAsset(assets: AssetWithMeta[]) {
-  return assets.length === 1;
+function principalHasOnlyStxCryptoCurrencyAsset(assets: StacksFungibleTokenAssetBalance[]) {
+  return assets.length === 0;
```

Comment:
this is confusing, if there's zero, there's one?

Reply:
Yeah, I just renamed something bc now this is just checking ft asset array bc the native crypto asset it separate. I'll make it more clear.